### ISSUE TITLE
Lupos : Hunter's exploit on Northdale

### ISSUE THREAD

**Issue by Wifibee** [LightsHope/issues#531]
Lupos is a rare wolf that can be tamed by hunters and keep a particularity : dealing shadow damage as melee.
It was fixed later for Blizzard and i think actual BUGS or UNINTENTIONAL behavior of mechanics shouldn't be put as is in the patches and the phrase "just as retail" shouldn't be an excuse.
If bug exploits are punished, this is just another one.

**Comment by Gamemechanicwow** [comment]
Do you have any sources that confirms that this was bug or unintentional? Because this subject has been discussed many times before, and just calling it a bug without contributing with anything further is just a waste of time.

This is a progressive server and Lupos deals shadowdamage until patch 1.09, where pet attacks got normalized.

**Comment by Sylvatir** [comment]
Yet you still have the pet abilities that were only introduced in later patches from the start of the server?
This is a clear gameplay mechanic issue and if the gameplay is going to use patch 1.12.1 so should this "feature". If the enemy NPC has 40% damage reduction from armor, this pet will deal around 70% more damage than all other pets. Damage that is scaling depending on the gear of the hunter. Do you really think this issue isn't gamebreaking?

**Comment by Evgenmaga** [comment]
Hey. I believe that the entire path of the WoW patches led us to the balance of pvp and pve games on servers. The developers faced problems and bugs on the 2004 classic warcraft servers. By the way, at that time there was not a lot of information on WoW, in particular on the hunter’s pets. The fact that Lupos violates pvp balance, few of the players knew. Now, after the incident of so many years, almost every player knows that this pet is imba. If you were on the server, you know that 90% of hunters use this wolf. Was it like a classic?

My answer is no. Is it normal? This is, as I said, the problem of information availability now.
By the way, if you are holding on to the "sequence of changes in the patches", then why are we now on version 1.6. use talents version 1.12? I am not against it, it just breaks your logic. I have a level 60 hunter, and I consider myself a very good player in pvp, but as if I did not want to, I always advocate balance in the game. Therefore, I will tell you so, this pet needs the abolition of dark damage. He can destroy 1 on 1 other players, even without the help of a hunter. Check it out, do the tests.

**Comment by Gamemechanicwow** [comment]
We are moving away from a bugreport to a discussion, and im not sure if this the best place to have this conversation. But.. here i go...
>I believe that the entire path of the WoW patches led us to the balance of pvp and pve games on servers

1.12.1 was never balanced in pve or pvp.
>Now, after the incident of so many years, almost every player knows that this pet is imba. If you were on the server, you know that 90% of hunters use this wolf. Was it like a classic?

The same goes for every other class. Everyone know the best rotations, BIS items etc.
>By the way, if you are holding on to the "sequence of changes in the patches", then why are we now on version 1.6. use talents version 1.12?

Limitations of not using progressive clients. This project is stuck on using spells(talents) available in the 1.12.1 client.
> Therefore, I will tell you so, this pet needs the abolition of dark damage. He can destroy 1 on 1 other players, even without the help of a hunter. Check it out, do the tests.

I ranked to 14 and Lupos wasn't the most unbalanced thing i saw on the journey. Lupos can't carry a bad player, but a good player can carry any class.

**Comment by Gamemechanicwow** [comment]
>Yet you still have the pet abilities that were only introduced in later patches from the start of the server?

We also have player spells and talent that were only introduced in later patches.

**Comment by Evgenmaga** [comment]
> Мы отходим от отчета об ошибках к обсуждению, и я не уверен, что это лучшее место для этого разговора. Но ... здесь я иду ...
> 
> > Я считаю, что весь путь WoW-патчей привел нас к балансу игр pvp и pve на серверах
> 
> 1.12.1 никогда не был сбалансирован в pve или pvp.
> 
> > Теперь, после стольких лет, почти каждый игрок знает, что это домашнее животное - имба. Если вы были на сервере, вы знаете, что 90% охотников используют этого волка. Это было похоже на классику?
> 
> То же самое касается и других классов. Все знают лучшие вращения, предметы BIS и т. Д.
> 
> > Кстати, если вы держитесь за «последовательность изменений в патчах», то почему мы сейчас находимся на версии 1.6. использовать таланты версии 1.12?
> 
> Ограничения не использования прогрессивных клиентов. Этот проект застрял в использовании заклинаний (талантов), доступных в клиенте 1.12.1.
> 
> > Поэтому я скажу вам, что это домашнее животное нуждается в отмене темного урона. Он может уничтожить 1 на 1 другого игрока, даже без помощи охотника. Проверьте это, выполните тесты.
> 
> Я занял 14 место, и Лупос не был самым неуравновешенным, что я видел в путешествии. Lupos не может нести плохого игрока, но хороший игрок может нести любой класс.


What can you say if you do not know what the word balance means. If you make a completely different effort to win a 1 in 1 battle, then this is an imbalance. If you have less chance of winning, even if you don’t let the hunter move away from you as a warrior, this is also called imbalance. You cannot kill a wolf in a 1 in 1 battle, although he is only an auxiliary element of the hunter class. You cannot control him as a magician if he uses rage. I do not need to say common words, prove the opposite. Say at once, you can not.

**Comment by Gamemechanicwow** [comment]
> What can you say if you do not know what the word balance means. If you make a completely different effort to win a 1 in 1 battle, then this is an imbalance.

So.. One guy puts more effort into his character than an other guy, and thats imbalance? I dont see your point. I was merely stating that any claims that 1.12.1 PVE/PVP is balanced is wrong.

>If you have less chance of winning, even if you don’t let the hunter move away from you as a warrior, this is also called imbalance.

Having a better weapon/item also increases chance of winning. Every class has BIS slots - hunters just got one more BIS slot to fill.

>You cannot kill a wolf in a 1 in 1 battle, although he is only an auxiliary element of the hunter class.

Thats not true. If it was, battlegrounds would look alot different.

>You cannot control him as a magician if he uses rage.

I am not sure what you trying to say here. Do you mean bestial wrath? Thats correct and it affects any type of hunter pet.

**Comment by Evgenmaga** [comment]
> > Что вы можете сказать, если не знаете, что означает баланс слова. Если вы сделаете совершенно разные усилия, чтобы выиграть битву 1 в 1, то это дисбаланс.
> 
> Итак ... Один парень прикладывает больше усилий к своему персонажу, чем к другому парню, и это несбалансированность? Я не вижу твоей точки. Я просто заявлял, что любые утверждения, что 1.12.1 PVE / PVP сбалансированы, ошибочны.
> 
> > Если у вас меньше шансов на победу, даже если вы не позволите охотнику отойти от вас как воина, это также называется дисбалансом.
> 
> Наличие лучшего оружия / предмета также увеличивает вероятность выигрыша. У каждого класса есть слоты BIS - у охотников только что появился еще один слот BIS.
> 
> > Вы не можете убить волка в битве 1 в 1, хотя он является лишь вспомогательным элементом класса охотников.
> 
> Это не правда. Если бы это было так, поля битвы выглядели бы совсем иначе.
> 
> > Вы не можете контролировать его как волшебника, если он использует ярость.
> 
> Я не уверен, что вы пытаетесь сказать здесь. Вы имеете в виду зверский гнев? Это правильно, и это влияет на любого типа охотника.

I see that for you it is difficult to understand, because You are not leveling your game character to level 60 now. Take the situation in a vacuum. Green Hunter vs Green Warrior at 45 level. In a situation where the hunter's pet, Lupos, you lose being a warrior almost 100% of the time, the hunter will not even need to move away from you to win, although he is in a situation that is obviously unfavorable to him.More than half of the damage will cause you a wolf hunter. Those. the hunter will not even need to make efforts to win, which cannot be said about a warrior.
In a situation where the hunter's pet is any other, your chances will be approximately equal, and if the warrior does not let the hunter retreat for shots, he will win. And vice versa. This is the essence of pvp balance, using the vulnerabilities of the enemy and the strengths of your class to win.
Bottom line: in the case of Lupos, you do not need to use tactics and skill, you just need to send your pet to the attack and beat you in melee.

**Comment by Gamemechanicwow** [comment]
Take a deep breath and try to read what wrote again.

>You are not leveling your game character to level 60 now

I have been part of the vanilla community so long, that i have seen this discussion 1000 times and I am not going to continue discussing this with you. Since you don't bring anything new and Lupus will not be changed anyway.

Take care.

**Comment by Evgenmaga** [comment]
> Take a deep breath and try to read what wrote again.
> 
> > You are not leveling your game character to level 60 now
> 
> I have been part of the vanilla community so long, that i have seen this discussion 1000 times and I am not going to continue discussing this with you. Since you don't bring anything new and Lupus will not be changed anyway.
> 
> Take care.

I will wait for your apologies here immediately after the launch of the official classic version in the summer of 2019. I think that the developers will not leave this defect with the pet at the time of release.
